Eplant/main.tsx

```diff
@@ -71,11 +71,11 @@ const registerServiceWorker = async () => {
     try {
       const registration = await navigator.serviceWorker.register(import.meta.env.BASE_URL + '/sw.js')
       if (registration.installing) {
-        console.log('Service worker installing')
+        // console.log('Service worker installing')
```

Comment:
Here too. Perhaps we can remove this? I'm not sure why we are keeping it as a comment